### DISCUSSION TITLE
Custom OPENAI URL on Dalle

### DISCUSSION THREAD

**Discussion by constLiakos** [open-webui/open-webui#1699] [General]
# Bug Report

## Description

**Bug Summary:**
In line 333 in config.py "OPENAI_API_BASE_URL = "https://api.openai.com/v1" command, prevents dall-e image genertion tool from using the custom openai base url. 
I am using LiteLLM Proxy for llm models and dall-e-3 generation for images.

## Environment

- **Open WebUI Version:**  0.1.120
- **Operating System:** Linux

## Reproduction Details

**Confirmation:**

- [x ] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.

## Note

Big fan of your project! Thank you in advance.

**Comment by tjbck** [comment]
I believe we already have this functionality, just not exposed to the frontend UI. I'll update it on our dev branch soon!

**Comment by tjbck** [comment]
Updated the code on our dev branch, let me know if that covers your use case!

**Comment by Komorebi-yaodong** [comment]
I get the error when using dalle3: Something went wrong :/ 'b64_json'

**Comment by zyj20200** [comment]
I also have the same problem：[ERROR: 'b64_json']

**Reply by Komorebi-zyd** [reply]
I solved it. Need your dalle's api supports base64 format for returning results
